.gitignore

```diff
@@ -4,5 +4,6 @@ pull_locales_login.rb
 node_modules/
 yarn.lock
 *.zip
+*.log
```

Comment:
Why? Do we have something that generates log files?

Reply:
I got some logs while debugging android browsers. Do you want to revert this change?

Reply:
I can't ignore all log files though I need them in other repos.

Reply:
Well, then `.git/info/exclude` in the repo.